cola/compat.py

```diff
@@ -14,7 +14,12 @@
 
 try:
     # pylint: disable=bytes-builtin
-    bstr = bytes
+    bytes()
+    if PY3:
+        def bstr(x):
+            return bytes(x, encoding='latin1')
```

Comment:
this is starting to look suspiciously like `core.encode()`.  Shouldn't it be `utf8`?